python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -854,82 +854,64 @@ def get_parent_asset_partitions_updated_after_child(
             )
         return updated_parents
 
-    def get_outdated_ancestors(
-        self, *, asset_partition: AssetKeyPartitionKey
-    ) -> AbstractSet[AssetKey]:
-        """Return the set of assets that are ancestors of the given asset partition and have parents
-        that have been updated more recently than they have.
+    def have_ignorable_partition_mapping_for_outdated_check(
+        self, asset_key: AssetKey, in_asset_key: AssetKey
```

Comment:
Is "in" instead of "upstream" because this goes both directions? if so, would use "from" / "to" to be consistent with how we talk about this in the world of partition mapping?

Reply:
I just made it "in" to match the signature of "get_partition_mapping"

Reply:
I think "upstream" is fine as well